ale/base/label_pds3.py

```diff
@@ -25,92 +15,202 @@ def label(self):
                 raise ValueError("{} is not a valid label".format(self._file))
         return self._label
 
-    @property
-    def _focal_plane_tempature(self):
-        return self.label['FOCAL_PLANE_TEMPERATURE'].value
 
     @property
     def instrument_id(self):
-        pass
+        """
+          Returns
+        -------
+        : str
+          Short name of the instrument
+        """
+        return self.label['INSTRUMENT_ID']
+
+
+    @property
+    def instrument_name(self):
+        """
+          Returns
+        -------
+        : str
+          Full name of the instrument
+        """
+        return self.label['INSTRUMENT_NAME']
+
+
+    @property
+    def instrument_host_id(self):
```

Comment:
Do we want to expose these hot keys?

Reply:
@Kelvinrr please explain

Reply:
basically, do we want to prefix it with an underscore or not. Without it, it is considered part of the public API.

Reply:
No problem one way or the other. What reasons do we have to hide the keys?

Reply:
Honestly, i'm not entirely sure. If this is specific to PDS3, then maybe hide the particular one?